### ISSUE TITLE
RuntimeError: Error building extension '_splat'

### ISSUE THREAD

**Issue by hongge831** [dolev104/neural_congealing#4]
nice work, I encontered this erro, could you please help me to figure it out?

**Comment by dolev104** [comment]
Hi @hongge831,

Thank you for your interest in our work.
This issue has to do with the splatting algorithm. We used GANgealing's splatting implementation.
You may find an answer [here](https://github.com/wpeebles/gangealing/issues/23).

**Comment by morrisalp** [comment]
I found this issue could be avoided by setting `use_griddata: True` in the training config file.

**Comment by dolev104** [comment]
Yes, we also added this option in the code. Just note that it is slower :)
